roles/setup_repo/tasks/setup_repo.yml

```diff
@@ -36,14 +39,23 @@
 
 - name: Cleanup the repos based on force_install
   block:
-    - include_tasks: "{{ pg_type }}_rhel_rm_repos.yml"
+    - include_tasks: "PG_RedHat_rm_repos.yml"
```

Comment:
You should add a condition to restrict this task for RedHat based OS

Reply:
FIxed. Changed it to something like:
`PG_{{ ansible_os_family }}_rm_repos`